conf/sample.ini

```diff
@@ -398,6 +398,9 @@
 ;ldap_sync_ttl = 60
 ;whitelist = 192.168.1.1, 192.168.2.1
 ;headers = Email:X-User-Email, Name:X-User-Name
+# If you want to combine auth proxy with the normal login session token & cookie enable setting defined below.
+# It will grant a login token if you supply auth proxy headers on /login request
```

Comment:
Shouldn't this read if you only provide auth proxy headers for the /login request? Quite hard to explain all of the details in a couple of lines here. Maybe reference to check documentation instead or/and?